src/librc/meson.build

```diff
@@ -1,11 +1,6 @@
 rc_h_conf_data = configuration_data()
-if rootprefix == '/'
-  rc_h_conf_data.set('PREFIX', '')
-  else
-  rc_h_conf_data.set('PREFIX', rootprefix)
-	endif
-rc_h_conf_data.set('LIB', libname)
-rc_h_conf_data.set('LIBEXECDIR', rc_libexecdir)
+rc_h_conf_data.set('RC_LIBEXECDIR', get_option('prefix') / get_option('libexecdir') / 'rc')
+rc_h_conf_data.set('RC_PLUGINDIR', get_option('prefix') / libdir / 'rc/plugins' )
```

Comment:
we probably should move plugins from `libdir` to `rc_libexecdir / 'plugins'`

or update the pkg-config to point to `lib` instead, but i'd rather have plugins in libexec